config/hubs/pangeo-181919.cluster.yaml

```diff
@@ -75,3 +68,11 @@ hubs:
               memory:
                 request: 1G
                 limit: 2G
+      grafana:
```

Comment:
This should be under `support.config`, since the grafana chart is a subchart of the support chart, not a hub template